querydsl-docs/src/main/docbook/en-US/content/tutorials/sql.xml

```diff
@@ -1203,5 +1203,53 @@ configuration.registerNumeric(5,2,Float.class);
 
   </sect2>
 
+  <sect2>
+
+    <title>Spring integration</title>
+
+    <para>Querydsl SQL integrates with Spring through the querydsl-sql-spring module:</para>
+
+    <programlisting language="xml"><![CDATA[
+<dependency>
+  <groupId>com.querydsl</groupId>
+  <artifactId>querydsl-sql-spring</artifactId>
+  <version>${querydsl.version}</version>
+</dependency>
+]]></programlisting>
+
+    <para>It provides Spring exception translation and a Spring connection provider for usage of Querydsl SQL
+    with Spring transaction managers. Below is a configuration example:</para>
+
+    <programlisting language="java"><![CDATA[
+    @Configuration
+    public class JdbcConfiguration {
+
+        // ...
+
+        @Bean
+        public PlatformTransactionManager transactionManager() {
+            return new DataSourceTransactionManager(dataSource());

```

Comment:
On one hand, you specifiy a `dataSource()`, so it would be wise to add:

```java

   @Bean
   public DataSource dataSource() {
      // implementation omitted
   }
```
Just for the sake of copy and pasting.